src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/CreateNamedProjectStep.ts

```diff
@@ -38,7 +38,7 @@ const AzureCostWarnings = () => {
     div({ style: rowEntry }, [
       icon('warning-standard', { size: 16, style: { marginRight: '0.5rem', color: colors.warning() } }),
       div([
-        'Creating a Terra on Azure Billing Project has a starting cost of $10.58/day and increases with usage. ',
+        'A Terra on Azure Billing Project has a starting cost of $10-15/day depending on region. ',
```

Comment:
oops, actually, can we say: "A Terra on Azure Billing Project has a starting cost of $10-15/day, depending on the region you select, and increases with usage."